src/nxdoc/web-ui/nuxeo-elements/nuxeo-elements-tutorial.md

```diff
@@ -30,7 +30,6 @@ Watch the related courses on Nuxeo University:</br>
 ## Requirements
 
 - **[Node.js](https://nodejs.org/)** is a JavaScript runtime built on Chrome's V8 JavaScript engine. Almost every tool out there for client-side development is built with Node.js and distributed with npm, the package manager for Node.js. Make sure you download and install your OS-specific version first (last verified versions for this tutorial are node v10.12.0 and npm 6.4.1)
-- **[Bower](http://bower.io/)** is currently **the** tool for managing web application definition.
 - **[Nuxeo CLI]({{page page='nuxeo-cli'}})** to scaffold your application **as a Nuxeo Bundle** and deploy it on a Nuxeo Server. Your application is **hosted inside** the Nuxeo Server as a bundle and uses it as a backend (last verified version for this tutorial is 1.9.0)
```

Comment:
The latest version of nuxeo cli is 1.9.2. Did you test with it?

Reply:
yes, I will update it to that version.